src/Ryujinx.Input/HLE/NpadController.cs

```diff
@@ -487,38 +487,35 @@ private static Vector3 Truncate(Vector3 value, int decimals)
             return value;
         }
 
-        public KeyboardInput? GetHLEKeyboardInput()
+        public static KeyboardInput? GetHLEKeyboardInput(IGamepadDriver KeyboardDriver)
```

Comment:
```suggestion
        public static KeyboardInput GetHLEKeyboardInput(IGamepadDriver KeyboardDriver)
```
If we never return null there is no need to keep the nullable modifier here.

Reply:
That is true, I'm not a keen to how C# works :D, I program in C++ primarily, Thank you for the point.